pyanaconda/modules/payloads/source/live_os/initialization.py

```diff
@@ -96,6 +100,22 @@ def __init__(self, image_path, target_mount):
         super().__init__(target_mount)
         self._image_path = image_path
 
+    def run(self):
+        """Run the task.
+
+        Calculates size of the live image image.
+        """
+        super().run()
+
+        required_space = self._calculate_required_space()
+        log.debug("Size of live OS image payload: %s.", Size(required_space))
+
+        return SetupLiveOSResult(required_space=required_space)
+
+    def _calculate_required_space(self):
+        source = os.statvfs(self._target_mount)
+        return source.f_frsize * (source.f_blocks - source.f_bfree)
```

Comment:
Have you compared the results with the previous solution? Does it return a similar size?

Reply:
Before the size estimate started approximately 440 MiB higher than than the new one and just by opening the storage spoke two more times with nearly no waiting it can grow another 80 MiB. 
I tried using this patch with one of rawhide ISOs and it installed fine.